src/SearchButton.cc

```
@@ -32,22 +32,29 @@ SearchButton::SearchButton(Decoration *decoration, const int buttonIndex, QObjec
+    painter->drawEllipse(circleRect);
 
+    const qreal sqrt2 = std::sqrt(2.0);
+    const qreal radius = 4.0;
     const qreal handleLength = 5.0;
-    const qreal sqrt2 = qSqrt(2);
-    const qreal handleAttachOffset = circleRadius / sqrt2;
-    const QPointF handleStart = circleCenter + QPointF(handleAttachOffset, handleAttachOffset);
-    const qreal handleEndOffset = (circleRadius + handleLength) / sqrt2;
-    const QPointF handleEnd = circleCenter + QPointF(handleEndOffset, handleEndOffset);
+
+    const QPointF circleCenter = circleRect.center();
```

Clang (14.0.6)

[warning] variable 'circleCenter' is not initialized

(cppcoreguidelines-init-variables)